crates/resolver-tests/src/lib.rs

```diff
@@ -182,7 +178,7 @@ pub fn resolve_with_config_raw(
 
     // The largest test in our suite takes less then 30 sec.
     // So lets fail the test if we have ben running for two long.
-    assert!(start.elapsed() < slow_cpu_multiplier(60));
+    assert!(start.elapsed().as_secs() < 60);
```

Comment:
Why remove `slow_cpu_multiplier`? Do we think that the other people (#6491) running Cargos CI will not run these tests?

Reply:
This was just a big chunk of code to migrate that didn't really seem worth it. The workaround can always be re-added if still necessary.